streaming_form_data/targets.py

```diff
@@ -84,6 +84,44 @@ def value(self):
         return b"".join(self._values)
 
 
+class ListTarget(BaseTarget):
+    """ValueTarget stores the input in an in-memory list of bytes.
+    This is useful in case you'd like to have the value contained in an
+    in-memory string.
+    """
```

Comment:
I'm slightly confused by the docstring here. According to the docstring, it seems like `ListTarget` stores the value in an in-memory string while `ValueTarget` uses an in-memory list of bytes. Isn't it the other way around? 🤔